code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -138,10 +138,28 @@
 
 /obj/item/reagent_containers/hypospray/medipen/examine()
 	. = ..()
-	if(reagents?.reagent_list.len)
-		. += span_notice("It is currently loaded.")
+	if(length(reagents?.reagent_list))
+		var/reagent_text = ""
+
+		if(length(reagents.reagent_list) == 1)
+			var/datum/reagent/first = reagents.reagent_list[1]
+			reagent_text = first.name
+
+		else
+			var/len = reagents.reagent_list.len
+			var/list/reagent_names_list = list()
+			for(var/i in 1 to (len - 2))
+				var/datum/reagent/reagent_object = reagents.reagent_list[i]
+				reagent_names_list += "[reagent_object.name], "
+			var/datum/reagent/last_reagent = reagents.reagent_list[len]
+			var/datum/reagent/second_to_last_reagent = reagents.reagent_list[len - 1]
+			reagent_names_list += "[second_to_last_reagent.name] and [last_reagent.name]"
+			reagent_text = reagent_names_list.Join("")
+
+		. += span_notice("Theres a small LCD screen on the side of the medipen which reads, 'WARNING: This medipen contains [reagent_text]. Do not use if allergic to any listed chemicals.' in small text.") //cheapskates at nanotrasen couldnt ball out for LED
```

Comment:
```suggestion
		. += span_notice("There's a small LCD screen on the side of the medipen which reads, 'WARNING: This medipen contains [reagent_text]. Do not use if allergic to any listed chemicals.' in small text.") //cheapskates at nanotrasen couldnt ball out for LED
```

Reply:
Oh I didn't know you could put apostrophes in descriptions, I thought it shmucked up like it does in announcements and such, thanks

Reply:
oh you might need to escape it yeah :flooshed: